CogniteSdk/src/Resources/Beta/Relationships.cs

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Cognite AS
+// SPDX-License-Identifier: Apache-2.0
+
+using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+
+using CogniteSdk.Beta;
+using static Oryx.Cognite.Beta.HandlerModule;
+using HttpContext = Oryx.Context<System.Net.Http.HttpResponseMessage>;
+
+namespace CogniteSdk.Resources.Beta
+{
+    // TODO: Move this out of namespace? CogniteSdk.Beta namespace is ignored when outside of namespace for some reason
```

Comment:
@dbrattli I couldn't figure out how to move the "using CogniteSdk.Beta;" out of this block. The same statement is on line 9 but has no effect. Any tips?